app/src/pages/inside/testCaseLibraryPage/emptyState/emptyState.jsx

```diff
@@ -24,6 +24,7 @@ import { referenceDictionary } from 'common/utils';
 
 import ImportIcon from 'common/img/import-thin-inline.svg';
 import { messages } from './messages';
+import { messages as parentMessages } from '../messages';
```

Comment:
Export multiple components for different empty state use cases from this component.

Reply:
Ok, it could be separate files in emptyState folder per each use case (we have at least 3 of them) and avoid this move of translations to parentMessages.